### PR TITLE
change_password2.0

### PR DESCRIPTION
**Pull request by yinxuan**

### CODE REVIEW

File: server/server/www/web/templates/change_password.html [hunk 1]
```diff
@@ -3,10 +3,51 @@
 <head>
     <meta charset="UTF-8">
     <title>{{ title }}</title>
+
+    <!-- 最新版本的 Bootstrap 核心 CSS 文件 -->
+<link rel="stylesheet" href="https://cdn.bootcss.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+<!-- 可选的 Bootstrap 主题文件（一般不用引入） -->
+<link rel="stylesheet" href="https://cdn.bootcss.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:11]: if optional, why import this?

File: server/server/www/web/templates/change_password.html [hunk 2]
```diff
@@ -3,10 +3,51 @@
...
+
+<!-- 最新的 Bootstrap 核心 JavaScript 文件 -->
+<script src="https://cdn.bootcss.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:14]: Why need this js file, it looks you didn't use any javascript.

File: server/server/www/web/templates/change_password.html [hunk 3]
```diff
@@ -3,10 +3,51 @@
...
+	</div>
+	<div style="text-align: center;font-size: 22px;"> 
+		Here is JianShi Change PassWord WEB!
+	</div>
+
+	<div class="container">
+		<div style="margin: 0 auto;">
+			<div style="padding: 30px;">
+				<textarea placeholder="Enter your email address" style="width: 100%"></textarea>
+				<div class="pull-right" style="box-shadow: 1,3,3,.2;padding: 5px;border-radius: 20px;background: #fab623;margin-top: 20px;">Seed password reset email</div>
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:32]: what's `pull-right`? delete it if you didn't define it in css file.

File: server/server/www/web/templates/change_password.html [hunk 4]
```diff
@@ -3,10 +3,51 @@
...
+				<!-- <button class="pull-right btn-primary" type="submit" style="margin-top: 20px;margin-bottom: 20px;">Seed password reset email</button>
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:33]: why don't delete it?

File: server/server/www/web/templates/change_password.html [hunk 5]
```diff
@@ -3,10 +3,51 @@
...
+ -->			</div>
+			<div style="padding: 30px;">
+				<textarea placeholder="Enter your email address" style="width: 100%"></textarea>
+				<div class="pull-right" style="box-shadow: 1,3,3,.2;padding: 5px;border-radius: 20px;background: #fab623;margin-top: 20px;">忘记密码</div>
+
+				<!-- <button class="pull-right" type="submit" style="margin-top: 20px;margin-bottom: 20px;background: #fab623">>忘记密码</button>
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:39]: why don't delete it?

File: server/server/www/web/templates/change_password.html [hunk 6]
```diff
@@ -3,10 +3,51 @@
...
+ -->			</div>
+			<div style="padding: 30px;">
+				<textarea placeholder="Enter your email address" style="width: 100%"></textarea>
+				<div class="pull-right" style="box-shadow: 1,3,3,.2;padding: 5px;border-radius: 20px;background: #fab623;margin-top: 20px;margin-bottom: 68px;">确认密码</div>
+
+				<!-- <button class="pull-right btn-primary" type="submit" style="margin-top: 20px;margin-bottom: 50px;">>确认密码</button>
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:45]: why don't delete it?

**Review by wingjay — CHANGES_REQUESTED** [review]

File: server/server/www/web/templates/change_password.html (hunk 2 above)
Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:14]: 以后会需要用到JS

File: server/server/www/web/templates/change_password.html (hunk 1 above)
Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:11]: 一些开关啥的控件和基本样式会需要用到的

File: server/server/www/web/templates/change_password.html (hunk 3 above)
Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:32]: pull-right类似float: right；是右浮动，BootStrap里定义好了的

File: server/server/www/web/templates/change_password.html (hunk 4 above)
Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:33]: 提供多种开关选择

**Pull request merged**